test/detector/test_options.py

```diff
@@ -113,23 +122,12 @@ def test_options_descriptor_set_not_existing(self):
             Options(
                 original_api_definition_dirs=None,
                 update_api_definition_dirs=None,
+                original_proto_files=None,
+                update_proto_files=None,
                 original_descriptor_set_file_path="fake_descriptor_set.pb",
                 update_descriptor_set_file_path="fake_descriptor_set.pb",
             )
 
-    def test_options_json_file_not_existing(self):
-        with self.assertRaises(TypeError):
-            with mock.patch("os.path.isdir") as mocked_isdir:
-                mocked_isdir.return_value = True
-                # The output json file is not existing, raise TypeError.
-                Options(
-                    original_api_definition_dirs="c,d",
-                    update_api_definition_dirs="a,b",
-                    original_descriptor_set_file_path=None,
-                    update_descriptor_set_file_path=None,
-                    output_json_path="not_existing.json",
-                )
-
```

Comment:
This test is removed because if 
1) use descriptor_set option, the descriptor_set_file_path requires `os.path.isfile` to be true 
2) use proto files directory and file path, the proto_files checking requires `os.path.isfile` to be true 
then we cannot test the type error when the custom json file is not existing.